expression/builtin_time.go

```diff
@@ -5370,3 +5401,19 @@ func (b *builtinLastDaySig) evalTime(row types.Row) (types.Time, bool, error) {
 	}
 	return ret, false, nil
 }
+
+func timePrecision(ctx sessionctx.Context, expression Expression) (int, error) {
```

Comment:
add a comment for this function.

Reply:
- - name is not very suitable, it was renamed and add comments in new commit.